.gitignore

```diff
@@ -36,6 +36,9 @@ coverage.xml
 .cache
 htmlcov
 
+# Benchmarks
+*.json
```

Comment:
I think this is too broad, in case someone is trying to add some json unrelated to benchmarking. Can do your other suggestion of `**/tests/*.json`?